detox/src/artifacts/utils/buildDefaultArtifactsRootDirpath.test.js

```diff
@@ -7,7 +9,7 @@ describe('buildDefaultArtifactsRootDirpath', () => {
   });
 
   it('should append subdir if dir does not end with /', () => {
-    expect(buildDefaultArtifactsRootDirpath('iphone8', 'artifacts')).toBe('artifacts/iphone8.Today 1:23:45PM');
+    expect(buildDefaultArtifactsRootDirpath('iphone8', 'artifacts')).toBe(path.join('artifacts', 'iphone8.Today 1:23:45PM'));
```

Comment:
paths/files can not include the character `:` or any punctuation mark for that matter.

Reply:
Why not? It is escaped by OS.